terra-jupyter-gatk-ovtf/tests/ovtf_smoke_test.ipynb

```diff
@@ -0,0 +1,95 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test cases requiring or benefiting from the context of a notebook\n",
+    "\n",
+    "If the notebook runs successfully from start to finish, the test is successful!\n",
+    "\n",
+    "TODO(all): Add additional tests and/or tests with particular assertions, as we encounter Python package version incompatibilities not currently detected by these tests.\n",
+    "\n",
+    "In general, only add test cases here that require the context of a notebook. This is because this notebook, as currently written, will abort at the **first** failure. Compare this to a proper test suite where all cases are run, giving much more information about the full extent of any problems encountered."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Package versions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 freeze"
```

Comment:
any particular reason we're testing `freeze` here?

Reply:
I think it's to just log all the pip packages for debugging info